lsp-mode.el

```diff
@@ -7322,7 +7324,10 @@ PROPS is list of triple (path value boolean?) where PATH is the path to the
 property; VALUE can be a literal value, symbol to be evaluated, or either a
 function or lambda function to be called without arguments; BOOLEAN? is an
 optional flag that should be non-nil for boolean settings, when it is nil the
-property will be ignored if the VALUE is nil."
+property will be ignored if the VALUE is nil.
+
+Example: `(lsp-register-custom-settings '((\"foo.bar.buzz.enabled\" t t)))'
+(note the double parentheses)"
```

Comment:
nit: We've already said this is list of triplet `(path value boolean?)`, so the note here may not needed

Reply:
Actually, this text, quoting `PROPS is list of triple (path value boolean?)` is the exact reason I was confused and couldn't make it work till I saw in the chat people provided examples. I read this text as: "PROPS is a list of triple, which you would write as (path value boolean?)".